ui/app/AppLayouts/Profile/popups/SyncDeviceCustomizationPopup.qml

```diff
@@ -63,6 +65,14 @@ StatusDialog {
                 d.saveNewName()
             }
         }
+        StatusInput {
+            id: idInput
+            Layout.fillWidth: true
+            label: qsTr("Installation ID")
+            input.edit.readOnly: true
+            text: root.deviceModel.installationId
+            visible: root.advancedStore.isDebugEnabled
```

Comment:
I think there is a `copy` mode you could use so that a "Copy to clipboard" button is added. That'd be a nice little QOL improvement

Reply:
Would be great, but I couldn't find such mode 🙁 
It seems that we duplicate smth like this everywhere:
https://github.com/status-im/status-desktop/blob/40fad7ac0b8dcb985bb6c0ca90f5c1bfb37fac34/ui/app/AppLayouts/Profile/popups/setupsyncing/DisplaySyncCode.qml#L201-L205

Probably would be good to implement it in the same way as `clearButton` here:
https://github.com/status-im/status-desktop/blob/40fad7ac0b8dcb985bb6c0ca90f5c1bfb37fac34/ui/StatusQ/src/StatusQ/Controls/StatusBaseInput.qml#L431-L441

But I'd leave it for now not to spend time, it doesn't seem that important, wdyt?

Reply:
Yeah it was just a suggestion. I think maybe it was the old button component that had it integrated. Anyway, feel free to merge as is